server/src/main/java/org/apache/iotdb/db/engine/tsfilemanagement/HotCompactionMergeTaskPoolManager.java

```diff
@@ -100,7 +104,8 @@ public ServiceType getID() {
     return ServiceType.HOT_COMPACTION_SERVICE;
   }
 
-  public void submitTask(HotCompactionMergeTask hotCompactionMergeTask) {
+  public void submitTask(HotCompactionMergeTask hotCompactionMergeTask)
+      throws RejectedExecutionException {
```

Comment:
RejectedExecutionException is a runtime exception. No need to throw it.

Reply:
yes